src/new/wallet.js

```diff
@@ -107,6 +112,10 @@ class HathorWallet extends EventEmitter {
       throw Error('You can\'t share connections.');
     }
 
+    if (multisig && !(multisig.pubkeys && multisig.minSignatures)) {
```

Comment:
Maybe we could assert that `pubkeys.length >= minSignatures`.

Reply:
Done in [89d3a89](https://github.com/HathorNetwork/hathor-wallet-lib/pull/324/commits/89d3a89fb831ed94006445e3d29178170df31c29)